compiler/modules/front/astString.bal

```diff
@@ -0,0 +1,95 @@
+import wso2/nballerina.err;
+
+type Code [-1|0|1]|string;
```

Comment:
* `Code` is not a good name - too easily confused with code as in codeGen.
* Why are you using a 1-tuple here?
* _IN/_OUT would be better as _INDENT/_OUTDENT

I suggest:

```
const LF_INDENT = 1;
const LF_OUTDENT = -1;
const LF = 0;

type Word string|LF_INDENT|LF_OUTDENT|LF;
```

Reply:
I was considering `Fragment` but decided against it because it's too long. Will use `Word`.
`const` approach looks good for the definition, the downside is we'll need extra casts due to jBal shortcomings, but we shouldn't base our design on that. Will change.